src/checkpoints.cpp

```diff
@@ -24,12 +24,14 @@ namespace Checkpoints
     //
     static MapCheckpoints mapCheckpoints =
         boost::assign::map_list_of
-        ( 0,	hashGenesisBlockOfficial )
+        ( 0, hashGenesisBlockOfficial )
+        ( 358278, uint256("0x02271ac3ba17a0295e376ab7a60e976cab81d6e1c616d1504059d0f43519d629"))
+        ( 703337, uint256("0xab0dad4b10d2370f009ed6df6effca1ba42f01d5070d6b30afeedf6463fbe7a2"))
```

Comment:
For 703337, hash is wrong. Should be: e0aba8843b4777f87d30369b115c5d4942bbbf062d5774880ef893d85643f42c